src/client/input/sdl.c

```diff
@@ -167,12 +167,25 @@ static qboolean gyro_active = false;
 
 // Gyro calibration
 static float gyro_accum[3];
-static unsigned int num_samples;
-
 static cvar_t *gyro_calibration_x;
 static cvar_t *gyro_calibration_y;
 static cvar_t *gyro_calibration_z;
 
+#if SDL_VERSION_ATLEAST(2, 0, 14)	// support for controller sensors (gyro, accelerometer)
+
+static unsigned int num_samples;
+#define NATIVE_SDL_GYRO 1	// uses SDL_CONTROLLERSENSORUPDATE to read gyro
```

Comment:
''''c
#define NATIVE_SDL_GYRO
''''
Should be enough

Reply:
Thanks!